tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -6025,7 +6061,8 @@ void SPIRVEmitter::discoverAllCaseStmtInSwitchStmt(
     auto bitWidth = astContext.getIntWidth(caseExpr->getType());
     if (bitWidth != 32)
       emitError("Switch statement translation currently only supports 32-bit "
-                "integer case values.");
+                "integer case values",
```

Comment:
non-32bit integer case value in switch statement unimplemented

Reply:
Done